visualize-admin/visualization-tool
# Use trees in selects when hierarchy is present

## How to test

- Use bathing water with hierarchies (check draft when searching)
- In the data filters, the "observation location" select should display the hierarchy as a tree
- When activating interactive filters, the interactive filter also displays the hierarchy as a tree

## TODO

- [x] When you first open the select, it loads the hierarchy and then closes the select, and you have to open it again
